neuro_san/test/driver/data_driven_agent_test_driver.py

```diff
@@ -49,8 +49,6 @@ def __init__(self, asserts: AssertForwarder, fixtures: FileOfClass = None):
         """
         self.asserts_basis: AssertForwarder = asserts
         self.fixtures: FileOfClass = fixtures
-        if self.fixtures is None:
-            self.fixtures = FileOfClass(__file__, path_to_basis="../../fixtures")
```

Comment:
Don't bother with the FileOfClass() stuff at this level to find test cases.
This will export to other repos better as well.